src/lib/lessons.ts

```diff
@@ -1,116 +1,149 @@
 import {LessonResource} from 'types'
 import {getGraphQLClient} from '../utils/configured-graphql-client'
 import getAccessTokenFromCookie from '../utils/get-access-token-from-cookie'
+import {loadLessonComments} from './lesson-comments'
+import {sanityClient} from 'utils/sanity-client'
+import groq from 'groq'
 
-export async function loadLesson(slug: string, token?: string) {
-  const query = /* GraphQL */ `
-    query getLesson($slug: String!) {
-      lesson(slug: $slug) {
```

Comment:
Moved the query (the string itself) out of the function and down below. This makes it so that when you open this file you can read the whole `loadLesson` function at a glance to see what it is doing.